### ISSUE TITLE
rpqueue.run or rpqueue project not compatible with Python3?

### ISSUE THREAD

**Issue by mgraupner** [josiahcarlson/rpqueue#18]
Is it possible that the rpqueue.run method or the whole project is not compatible with Python3? I wanted to start the execution daemon and got the following error message:
```
File "/virtenv/lib/python3.6/site-packages/rpqueue/__init__.py", line 881, in execute_tasks
    for p in xrange(processes):
NameError: name 'xrange' is not defined
```
After switching to python2 the executor started as expected.
If so you should probably mention it somewhere in the documentation. Otherwise I would love to know why it does not work for me.

Thanks!

**Comment by josiahcarlson** [comment]
Wasn't intentional for this to not be Python 3 compatible. I've swapped the few examples of xrange with range, and the tests all seem to pass for Python 2.6, 2.7, 3.3, 3.4, 3.5, and 3.6. Let me know if you have any more difficulties.

**Comment by josiahcarlson** [comment]
P.S. This should be fixed in version 0.27.0

**Comment by mgraupner** [comment]
Thank you very much for fixing this. I will give it a try in the next days.
